src/main/java/com/google/sps/data/DataStoreManager.java

```diff
@@ -0,0 +1,342 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import java.lang.reflect.Type;
+import com.google.gson.reflect.TypeToken;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for creating new tasks. **/
+public class DatastoreManager implements DataManager {
+
+  private DatastoreService datastore = DatastoreServiceFactory.getDataStoreService();
+  Gson gson = new Gson();
+  
+  public DataStoreManager() {  
```

Comment:
This constructor can be deleted because it is equivalent to if there were no constructor at all.

Reply:
If the constructor is not used at all, then we should put it a private constructor like below:

private DataStoreManager() {}

Also, mark all other methods as static methods. Add the "static" keyword to all following methods.

public static void storeGame(Game game) {
}